sunpy/map/mapbase.py

```diff
@@ -154,12 +148,14 @@ class GenericMap(NDData):
       notation using equations 32 in Thompson (2006).
 
     * If a CDi_j matrix is provided it is assumed that it can be converted to a
-      PCi_j matrix and CDELT keywords as described in Greisen & Calabretta (2002).
+      PCi_j matrix and CDELT keywords as described in Greisen & Calabretta
+      (2002).
```

Comment:
In the process, I would  add the link: `http://dx.doi.org/10.1051/0004-6361:20021327`
and the one above for Thompson (2006): `http://dx.doi.org/10.1051/0004-6361:20054262`